Examples/restcomm-olympus-nopod/restcomm-olympus/LocalContact.h

```diff
@@ -0,0 +1,19 @@
+//
+//  LocalContact.h
+//  restcomm-olympus
+//
+//  Created by Manevski Ognjen on 9/5/17.
+//  Copyright © 2017 TeleStax. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface LocalContact : NSObject<NSCoding>
```

Comment:
Right, I was wondering if we could instead use standard NS containers instead that should be handled automatically, to avoid doing serialization/deserialization work ourselves. On the other hand it's good that now have an object per contact, so I'd say leave as is for now and don't spend too much time on the App for such details